network/cloudflared/manifest.yaml

```diff
@@ -0,0 +1,13 @@
+version: v1alpha1
+metadata:
+  name: cloudflared
+  version: "$VERSION"
+  author: Maxime Narbaud
+  description: |
+    Cloudflare Tunnel securely connects resources to Cloudflare without a public IP.
+    A lightweight daemon (cloudflared) creates outbound-only connections to Cloudflare,
+    allowing safe access to services like HTTP, SSH, remote desktops, and other protocols.
+    More info: https://github.com/cloudflare/cloudflared/
+  compatibility:
+    talos:
+      version: ">= v1.5.0"
```

Comment:
1.9.0

Reply:
I think from pure API perspective, 1.5.0 should be fine as well? (if someone builds manually?)

Reply:
yeh, will keep as is then